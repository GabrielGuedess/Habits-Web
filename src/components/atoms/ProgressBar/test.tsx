import { render } from '@testing-library/react';

import { ProgressBar } from '.';

describe('<ProgressBar />', () => {
  it('should render the ProgressBar', () => {
    const { container } = render(<ProgressBar progress={70} />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
