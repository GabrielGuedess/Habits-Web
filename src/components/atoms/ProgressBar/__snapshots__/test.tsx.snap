// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgressBar /> should render the ProgressBar 1`] = `
<div
  class="mt-4 h-3 w-full rounded-xl bg-zinc-700 "
>
  <div
    aria-label="Progresso de hÃ¡bitos"
    aria-valuenow="70"
    class="h-3 rounded-xl bg-violet-600"
    role="progressbar"
    style="width: 70%;"
  />
</div>
`;
